maxim/tests/test_livekit_realtime.py

```
@@ -0,0 +1,97 @@
+        Performs a web search for the given query.
+        """
+        if not TAVILY_API_KEY:
+            return "Tavily API key is not set. Please set the TAVILY_API_KEY environment variable."
+
+        tavily_client = TavilyClient(api_key=TAVILY_API_KEY)
+        try:
+            response = tavily_client.search(query=query, search_depth="basic")
+            if response.get("answer"):
+                return response["answer"]
+            return str(response.get("results", "No results found."))
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)